src/css/mobile-menu.css

```diff
@@ -96,6 +101,7 @@
   font-size: 14px;
   line-height: 18px;
   font-weight: 600;
+  color: var(--white);
```

Comment:
Please add hover effect for navigation item and for social icons

Reply:
done